README.md

```diff
@@ -25,7 +25,7 @@ If you are struggling to define a stakeholder, we recommend you complete a proje
 
 ### The Data
 
-This project uses the King County House Sales dataset, which can be found in  `kc_house_data.csv` in the data folder in this repo. The description of the column names can be found in `column_names.md` in the same folder. As with most real world data sets, the column names are not perfectly described, so you'll have to do some research or use your best judgment if you have questions about what the data means.
+This project uses the King County House Sales dataset, which can be found in  `kc_house_data.csv` in the data folder in this assignment's github repository. The description of the column names can be found in `column_names.md` in the same folder. As with most real world data sets, the column names are not perfectly described, so you'll have to do some research or use your best judgment if you have questions about what the data means.
```

Comment:
This change is specifically because "Where is the repository?" is a question I received from 3 students this week